README.md

```diff
@@ -24,6 +24,19 @@ This module includes a small demonstration in `index.js`.
 npm start
 ```
 
+To delete a feed, first select a valid ssb feed id, and run these two commands:
+
+```
+node ssb.js --id='feedId'
+node delete-views
+```
+
+It can also help to private block a feed, so that it doesn't resync later: 
+
+```
+scuttlebot publish --type contact --blocking --recps 'yourId' --contact 'feedId'
```

Comment:
Is there a benefit to doing this block privately rather than doing a public block?

Reply:
It's probably a matter of opinion, we could include both public and private examples.